java/client/src/test/java/glide/api/RedisClientCreateTest.java

```diff
@@ -78,8 +84,8 @@ public void createClient_errorOnConnectionThrowsExecutionException() {
         connectToRedisFuture.completeExceptionally(exception);
         RedisClientConfiguration config = RedisClientConfiguration.builder().build();
 
-        when(connectionManager.connectToRedis(eq(config))).thenReturn(connectToRedisFuture);
```

Comment:
why the change here?

Reply:
@SanHalacogluImproving you reverted this change in the previous test.  I think you can revert this change too.